src/System.Threading.Tasks.Dataflow/src/Base/DataflowBlockOptions.cs

```diff
@@ -170,6 +177,13 @@ public string NameFormat
                 _nameFormat = value;
             }
         }
+
+        /// <summary>Gets whether ordered processing should be enforced on a block's handling of messages.</summary>
```

Comment:
> "Gets or sets"

Done.

> Can you be a bit clearer in the docs

Done.

> I wonder if we should "invert" the option, i.e. AllowUnorderedProcessing or something like that?

I understand that preference.  And in general I agree with you.  However, in this case it felt like a double-negative.  Additionally, the library has been around for a while and ordering is effectively always on; this new option provides a way to effectively turn it off, hence defaulting to true and allowing it to be set to false.

That said, while I prefer the default of true in this case, I don't feel terribly strong about it and would be ok inverting it if we can come up with a good name for it and if others prefer the inversion.